src/xcode/ENA/ENA/Source/Scenes/ENSetting/ExposureNotificationSettingViewController.swift

```diff
@@ -231,3 +228,12 @@ private extension ENSettingModel.Content {
 		}
 	}
 }
+
+// MARK: ENStateHandler Updating
+extension ExposureNotificationSettingViewController: ENStateHandlerUpdating {
+	func updateEnState(_ state: ENStateHandler.State) {
```

Comment:
This method is not called when the exposurenotification settings was opened via SettingsviewController and thus the status is wrong in this case